src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/TransformationController.java

```diff
@@ -12,16 +12,23 @@
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.in.ExceptionRecord;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.output.CaseCreationDetails;
 import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.output.SuccessfulTransformationResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformer.D8FormToCaseTransformer;
 
 import javax.validation.Valid;
 import java.util.Collections;
+import java.util.Map;
 
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
 
 @Slf4j
 @RestController
 public class TransformationController {
 
+    private static final String CASE_TYPE_ID = "FINANCIAL_REMEDY";
+    private static final String EVENT_ID = "EVENT_ID";
```

Comment:
Why do we need these values?

Reply:
These are expected in the response to BSP. At the moment we have no idea what the "real" values actually are, but these should suffice for the POC while we wait to find out what the actual values should be